packages/spanner/system-test/spanner.js

```diff
@@ -2747,7 +2747,7 @@ var spanner = new Spanner(env);
       it('should accept an exact staleness', function(done) {
         var options = {
           readOnly: true,
-          exactStaleness: 4
+          exactStaleness: Math.ceil((Date.now() - records[2].timestamp) / 1000)
```

Comment:
`4` meant "4 seconds ago". That might work when this test is run in isolation, but not when this describe suite runs all of the tests serially. I used this calculation to get a relative staleness value.